<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointsSamplingDialog</class>
 <widget class="QWidget" name="PointsSamplingDialog">
  <property name="windowTitle">
   <string>Points Sampling on mesh</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <property name="horizontalSpacing">
    <number>32</number>
   </property>
   <property name="verticalSpacing">
    <number>20</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 51, 54);</string>
     </property>
     <property name="text">
      <string>Sampling mode</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="pnRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(52, 51, 54);</string>
       </property>
       <property name="text">
        <string>Percentage</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>9999</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(52, 51, 54);</string>
       </property>
       <property name="text">
        <string>Grid</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 51, 54);</string>
     </property>
     <property name="text">
      <string>Sampling percentage</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="pnSpinBox">
     <property name="minimumSize">
      <size>
       <width>394</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>394</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QDoubleSpinBox
{
	color: #ffffff;
	background-color: #343336;
	border:1px solid #989898;
	padding-left:9px;
	border-radius: 2px;
	min-height:32px;
	max-height:32px;
}
QDoubleSpinBox:disabled
{
	color: #999999;
	background-color: #3F3F3F;
}</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>80.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
